_rules/inline_link_in_paragraph_is_distinguishable_be4d0c.md

```diff
@@ -0,0 +1,252 @@
+---
+id: be4d0c
+name: Inline link is distinguishable
+rule_type: atomic
+
+description: |
+  This rule checks that links that are embedded in a paragraph, list or cell have a way to distinguish them as links compared to the rest of the text
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+acknowledgements:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any HTML element which:
+
+- has the [semantic role][] of ['link'][link] or a [semantic role][] that inherits from the ['link'][link] role; and
+- is a [descendant][] of a `p`, `li` or `td` HTML elements (henceforth referred to as the _ancestor_ element) in the [flat tree][]; and
+- has [descendant][] [text nodes][text node] in the [flat tree][] that are [visible][]; and
+- the _ancestor_ element has other [descendant][] [visible][] [text nodes][text node] in the [flat tree][].
```

Comment:
I don't think something like this should be applicable:

```html
<p>
  Hello world
  <br><br>
  <a href="">Next page</a>
</p>
```

Reply:
Agree. But the `<br>` element should not be used like that according to the [HTML spec](https://html.spec.whatwg.org/#the-br-element). The spec says:
- `br` elements must be used only for line breaks that are actually part of the content, as in poems or addresses.
- `br` elements must not be used for separating thematic groups in a paragraph (which is what you are doing in that example)

I don't think we consider incorrect uses of HTML elements in other rules, so I don't think we need to do that in this rule, and I won't change the applicability based on this (unless there are other arguments).